molgenis/molgenis-emx2
# fix: issue that roles are shared between schema with equals names if uppercased.

fix: schema roles should be case sensitive to prevent permissions to be shared between schema with equal uppercase names. closes #405

To reproduce problem:

1. in PREVIOUS version create two schemas with same name, e.g. add 'Pet store' next to existing 'pet store'

2. see the problem that 'Pet store' has same members as 'pet store' while you didn't create it

Added migration. To test if that works

3. check, and see roles with "MG_ROLE_PET STORE" having uppercase while schema is "pet store"
using 'psql -u molgenis' and then 'SELECT rolname FROM pg_roles;'

4. run THIS version, which should then run the migration
gradle run

5. check, and see roles with "MG_ROLE_pet store" having same case as schema names
using 'psql -u molgenis' and then 'SELECT rolname FROM pg_roles;'
